packages/perseus/src/widgets/interactive-graphs/utils.ts

```diff
@@ -112,6 +112,10 @@ type ParsedNode = {
 function condenseTextNodes(nodes: Array<ParsedNode>): Array<ParsedNode> {
     const result: ParsedNode[] = [];
 
+    if (!nodes) {
```

Comment:
According to the types, this condition should never be met, because `nodes` is an array, and all arrays are truthy. I think we need to change the `nodes` parameter type to something like `ParsedNode[] | undefined` to match reality.